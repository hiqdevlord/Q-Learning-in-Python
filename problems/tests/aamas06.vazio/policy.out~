row1col1 ---
row1col2 West
row1col3 West
row1col4 West
row1col5 West
row1col6 West
row1col7 West
row1col8 West
row1col9 South
row1col10 South
row1col11 West
row1col12 West
row1col13 West
row1col14 West
row1col15 South
row1col16 South
row1col17 East
row1col18 East
row1col19 South
row1col20 South
row1col21 South
row1col22 West
row2col1 North
row2col2 North
row2col3 North
row2col4 West
row2col5 West
row2col6 North
row2col7 East
row2col8 South
row2col9 North
row2col10 North
row2col11 South
row2col12 East
row2col13 East
row2col14 North
row2col15 West
row2col16 East
row2col17 West
row2col18 North
row2col19 North
row2col20 West
row2col21 West
row2col22 North
row3col1 North
row3col2 North
row3col3 West
row3col4 West
row3col5 South
row3col6 West
row3col7 East
row3col8 East
row3col9 South
row3col10 East
row3col11 North
row3col12 South
row3col13 South
row3col14 South
row3col15 North
row3col16 East
row3col17 North
row3col18 West
row3col19 North
row3col20 South
row3col21 East
row3col22 South
row4col1 North
row4col2 North
row4col3 North
row4col4 North
row4col5 West
row4col6 East
row4col7 West
row4col8 West
row4col9 East
row4col10 South
row4col11 East
row4col12 North
row4col13 East
row4col14 South
row4col15 South
row4col16 North
row4col17 East
row4col18 East
row4col19 West
row4col20 North
row4col21 East
row4col22 North
row5col1 North
row5col2 West
row5col3 North
row5col4 West
row5col5 North
row5col6 West
row5col7 West
row5col8 North
row5col9 North
row5col10 West
row5col11 South
row5col12 East
row5col13 North
row5col14 West
row5col15 East
row5col16 South
row5col17 West
row5col18 South
row5col19 North
row5col20 South
row5col21 West
row5col22 South
row6col1 North
row6col2 North
row6col3 North
row6col4 North
row6col5 North
row6col6 North
row6col7 North
row6col8 West
row6col9 East
row6col10 West
row6col11 East
row6col12 North
row6col13 South
row6col14 West
row6col15 North
row6col16 East
row6col17 South
row6col18 North
row6col19 East
row6col20 West
row6col21 South
row6col22 South
row7col1 North
row7col2 West
row7col3 East
row7col4 North
row7col5 North
row7col6 South
row7col7 South
row7col8 South
row7col9 West
row7col10 East
row7col11 West
row7col12 West
row7col13 South
row7col14 North
row7col15 East
row7col16 West
row7col17 South
row7col18 East
row7col19 North
row7col20 East
row7col21 West
row7col22 North
row8col1 North
row8col2 West
row8col3 North
row8col4 West
row8col5 West
row8col6 South
row8col7 South
row8col8 West
row8col9 West
row8col10 South
row8col11 East
row8col12 West
row8col13 West
row8col14 West
row8col15 North
row8col16 West
row8col17 South
row8col18 West
row8col19 North
row8col20 North
row8col21 North
row8col22 West
row9col1 North
row9col2 East
row9col3 North
row9col4 West
row9col5 West
row9col6 West
row9col7 West
row9col8 West
row9col9 West
row9col10 West
row9col11 East
row9col12 East
row9col13 North
row9col14 East
row9col15 East
row9col16 West
row9col17 East
row9col18 East
row9col19 North
row9col20 East
row9col21 West
row9col22 North
row10col1 North
row10col2 North
row10col3 North
row10col4 North
row10col5 South
row10col6 North
row10col7 North
row10col8 North
row10col9 North
row10col10 West
row10col11 South
row10col12 South
row10col13 East
row10col14 South
row10col15 East
row10col16 North
row10col17 West
row10col18 West
row10col19 South
row10col20 North
row10col21 West
row10col22 South
row11col1 North
row11col2 South
row11col3 North
row11col4 North
row11col5 West
row11col6 West
row11col7 South
row11col8 East
row11col9 North
row11col10 West
row11col11 North
row11col12 South
row11col13 West
row11col14 North
row11col15 North
row11col16 South
row11col17 West
row11col18 East
row11col19 South
row11col20 East
row11col21 West
row11col22 West
row12col1 East
row12col2 West
row12col3 North
row12col4 North
row12col5 West
row12col6 South
row12col7 West
row12col8 South
row12col9 West
row12col10 West
row12col11 South
row12col12 East
row12col13 South
row12col14 East
row12col15 East
row12col16 South
row12col17 North
row12col18 West
row12col19 West
row12col20 East
row12col21 West
row12col22 South
row13col1 East
row13col2 South
row13col3 North
row13col4 North
row13col5 East
row13col6 North
row13col7 West
row13col8 North
row13col9 West
row13col10 North
row13col11 West
row13col12 South
row13col13 West
row13col14 East
row13col15 South
row13col16 West
row13col17 West
row13col18 East
row13col19 North
row13col20 South
row13col21 South
row13col22 West
row14col1 East
row14col2 West
row14col3 North
row14col4 West
row14col5 West
row14col6 South
row14col7 East
row14col8 North
row14col9 South
row14col10 South
row14col11 South
row14col12 West
row14col13 South
row14col14 South
row14col15 West
row14col16 West
row14col17 North
row14col18 South
row14col19 East
row14col20 North
row14col21 South
row14col22 North
row15col1 North
row15col2 South
row15col3 South
row15col4 South
row15col5 North
row15col6 South
row15col7 West
row15col8 West
row15col9 South
row15col10 West
row15col11 South
row15col12 East
row15col13 South
row15col14 West
row15col15 West
row15col16 East
row15col17 East
row15col18 South
row15col19 South
row15col20 South
row15col21 North
row15col22 South
row16col1 South
row16col2 South
row16col3 East
row16col4 West
row16col5 East
row16col6 North
row16col7 South
row16col8 North
row16col9 East
row16col10 West
row16col11 North
row16col12 South
row16col13 East
row16col14 East
row16col15 South
row16col16 West
row16col17 West
row16col18 West
row16col19 South
row16col20 West
row16col21 South
row16col22 North
row17col1 East
row17col2 South
row17col3 East
row17col4 South
row17col5 West
row17col6 East
row17col7 East
row17col8 South
row17col9 North
row17col10 South
row17col11 North
row17col12 West
row17col13 West
row17col14 South
row17col15 East
row17col16 North
row17col17 West
row17col18 South
row17col19 East
row17col20 South
row17col21 South
row17col22 West
row18col1 North
row18col2 South
row18col3 North
row18col4 South
row18col5 North
row18col6 North
row18col7 West
row18col8 West
row18col9 North
row18col10 North
row18col11 South
row18col12 North
row18col13 North
row18col14 East
row18col15 South
row18col16 North
row18col17 South
row18col18 South
row18col19 West
row18col20 West
row18col21 South
row18col22 North
row19col1 North
row19col2 West
row19col3 North
row19col4 East
row19col5 North
row19col6 West
row19col7 North
row19col8 East
row19col9 East
row19col10 West
row19col11 North
row19col12 East
row19col13 East
row19col14 North
row19col15 West
row19col16 West
row19col17 West
row19col18 West
row19col19 North
row19col20 East
row19col21 North
row19col22 North
