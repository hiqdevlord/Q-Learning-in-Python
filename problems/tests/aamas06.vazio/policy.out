row1col1 ---
row1col2 West
row1col3 West
row1col4 West
row1col5 West
row1col6 West
row1col7 West
row1col8 West
row1col9 West
row1col10 West
row1col11 West
row1col12 West
row1col13 West
row1col14 West
row1col15 West
row1col16 West
row1col17 West
row1col18 West
row1col19 West
row1col20 West
row1col21 West
row1col22 West
row2col1 North
row2col2 North
row2col3 North
row2col4 West
row2col5 West
row2col6 North
row2col7 North
row2col8 North
row2col9 West
row2col10 West
row2col11 North
row2col12 West
row2col13 North
row2col14 West
row2col15 West
row2col16 West
row2col17 North
row2col18 North
row2col19 West
row2col20 North
row2col21 West
row2col22 North
row3col1 North
row3col2 West
row3col3 North
row3col4 West
row3col5 West
row3col6 North
row3col7 North
row3col8 West
row3col9 North
row3col10 West
row3col11 North
row3col12 North
row3col13 North
row3col14 West
row3col15 West
row3col16 West
row3col17 West
row3col18 West
row3col19 North
row3col20 West
row3col21 North
row3col22 West
row4col1 North
row4col2 North
row4col3 West
row4col4 North
row4col5 West
row4col6 North
row4col7 West
row4col8 North
row4col9 West
row4col10 West
row4col11 West
row4col12 North
row4col13 North
row4col14 West
row4col15 West
row4col16 West
row4col17 North
row4col18 North
row4col19 West
row4col20 West
row4col21 North
row4col22 West
row5col1 North
row5col2 West
row5col3 West
row5col4 North
row5col5 West
row5col6 North
row5col7 North
row5col8 West
row5col9 North
row5col10 North
row5col11 North
row5col12 West
row5col13 North
row5col14 West
row5col15 West
row5col16 North
row5col17 North
row5col18 North
row5col19 North
row5col20 West
row5col21 North
row5col22 North
row6col1 North
row6col2 North
row6col3 North
row6col4 North
row6col5 North
row6col6 North
row6col7 North
row6col8 West
row6col9 West
row6col10 North
row6col11 North
row6col12 West
row6col13 West
row6col14 North
row6col15 North
row6col16 West
row6col17 West
row6col18 West
row6col19 West
row6col20 North
row6col21 West
row6col22 West
row7col1 North
row7col2 North
row7col3 North
row7col4 North
row7col5 West
row7col6 West
row7col7 North
row7col8 North
row7col9 North
row7col10 North
row7col11 North
row7col12 North
row7col13 North
row7col14 North
row7col15 West
row7col16 North
row7col17 West
row7col18 West
row7col19 West
row7col20 North
row7col21 West
row7col22 West
row8col1 North
row8col2 West
row8col3 North
row8col4 West
row8col5 North
row8col6 West
row8col7 North
row8col8 North
row8col9 North
row8col10 North
row8col11 North
row8col12 West
row8col13 West
row8col14 North
row8col15 West
row8col16 North
row8col17 West
row8col18 North
row8col19 North
row8col20 West
row8col21 West
row8col22 North
row9col1 North
row9col2 North
row9col3 West
row9col4 West
row9col5 North
row9col6 North
row9col7 West
row9col8 West
row9col9 West
row9col10 West
row9col11 West
row9col12 North
row9col13 North
row9col14 North
row9col15 North
row9col16 North
row9col17 West
row9col18 West
row9col19 West
row9col20 North
row9col21 West
row9col22 West
row10col1 North
row10col2 North
row10col3 West
row10col4 North
row10col5 West
row10col6 North
row10col7 North
row10col8 North
row10col9 North
row10col10 West
row10col11 North
row10col12 West
row10col13 West
row10col14 North
row10col15 North
row10col16 West
row10col17 West
row10col18 North
row10col19 West
row10col20 West
row10col21 West
row10col22 West
row11col1 North
row11col2 West
row11col3 West
row11col4 North
row11col5 North
row11col6 North
row11col7 North
row11col8 West
row11col9 North
row11col10 North
row11col11 West
row11col12 West
row11col13 North
row11col14 North
row11col15 North
row11col16 West
row11col17 West
row11col18 North
row11col19 North
row11col20 North
row11col21 West
row11col22 West
row12col1 North
row12col2 West
row12col3 West
row12col4 North
row12col5 West
row12col6 West
row12col7 North
row12col8 North
row12col9 West
row12col10 North
row12col11 West
row12col12 West
row12col13 West
row12col14 West
row12col15 North
row12col16 West
row12col17 West
row12col18 West
row12col19 West
row12col20 North
row12col21 West
row12col22 North
row13col1 North
row13col2 North
row13col3 North
row13col4 West
row13col5 North
row13col6 North
row13col7 West
row13col8 West
row13col9 West
row13col10 West
row13col11 West
row13col12 West
row13col13 North
row13col14 North
row13col15 North
row13col16 West
row13col17 West
row13col18 North
row13col19 West
row13col20 North
row13col21 West
row13col22 North
row14col1 North
row14col2 North
row14col3 West
row14col4 North
row14col5 North
row14col6 West
row14col7 West
row14col8 North
row14col9 North
row14col10 North
row14col11 West
row14col12 West
row14col13 North
row14col14 West
row14col15 West
row14col16 West
row14col17 North
row14col18 West
row14col19 West
row14col20 North
row14col21 West
row14col22 North
row15col1 North
row15col2 North
row15col3 West
row15col4 West
row15col5 West
row15col6 West
row15col7 West
row15col8 West
row15col9 West
row15col10 North
row15col11 West
row15col12 West
row15col13 West
row15col14 North
row15col15 West
row15col16 West
row15col17 North
row15col18 West
row15col19 West
row15col20 North
row15col21 West
row15col22 North
row16col1 North
row16col2 West
row16col3 West
row16col4 West
row16col5 West
row16col6 North
row16col7 North
row16col8 West
row16col9 West
row16col10 West
row16col11 West
row16col12 North
row16col13 West
row16col14 West
row16col15 North
row16col16 North
row16col17 North
row16col18 West
row16col19 North
row16col20 West
row16col21 North
row16col22 West
row17col1 North
row17col2 West
row17col3 North
row17col4 North
row17col5 North
row17col6 North
row17col7 North
row17col8 North
row17col9 West
row17col10 West
row17col11 North
row17col12 West
row17col13 West
row17col14 North
row17col15 West
row17col16 West
row17col17 North
row17col18 North
row17col19 North
row17col20 North
row17col21 North
row17col22 West
row18col1 North
row18col2 North
row18col3 West
row18col4 North
row18col5 West
row18col6 North
row18col7 North
row18col8 North
row18col9 North
row18col10 West
row18col11 North
row18col12 West
row18col13 West
row18col14 West
row18col15 West
row18col16 North
row18col17 North
row18col18 North
row18col19 North
row18col20 West
row18col21 North
row18col22 North
row19col1 North
row19col2 North
row19col3 West
row19col4 North
row19col5 North
row19col6 West
row19col7 North
row19col8 West
row19col9 North
row19col10 North
row19col11 West
row19col12 West
row19col13 West
row19col14 North
row19col15 West
row19col16 West
row19col17 North
row19col18 North
row19col19 North
row19col20 West
row19col21 West
row19col22 North
